html5lib/html5parser.py

```diff
@@ -901,7 +901,7 @@ def __init__(self, parser, tree):
                 ("iframe", self.startTagIFrame),
                 (("noembed", "noframes", "noscript"), self.startTagRawtext),
                 ("select", self.startTagSelect),
-                (("rp", "rt"), self.startTagRpRt),
+                (("rp", "rt", "rb", "rtc"), self.startTagRpRtRbRtc),
```

Comment:
This is wrong. You need two distinct groups. Or you need to also branch the `self.tree.openElements[-1].name != "ruby"` line further. What you're doing now doesn't match other implementations.